engine/access/rest/converter.go

```diff
@@ -171,9 +171,10 @@ func toSignature(signature string) ([]byte, error) {
 	if err != nil {
 		return nil, fmt.Errorf("invalid signature encoding")
 	}
-	if len(signatureBytes) != signatureLength {
-		return nil, errors.New("invalid signature length")
-	}
+	// TODO: fix the signature length validation
```

Comment:
not sure..but the validation currently fails. This will be addressed in a subsequent validation PR.

Reply:
I don't think anymore it would be even good to validate this, reasons for it is that the validation of signature is already implemented in deeper layers and length for now is for both hashing algos 256 bit that might change and I wouldn't like to do any more sanitisation of this input, I feel that the only reason to limit length is to not allow weird overflow attacks (which in any way I don't see it happening) but that we anyhow protected by setting payload size limit.